src/particle/particle/particle.py

```diff
@@ -1015,6 +1015,76 @@ def from_evtgen_name(cls: type[Self], name: str) -> Self:
         """
         return cls.from_pdgid(EvtGenName2PDGIDBiMap[name])
 
+    @classmethod
+    def from_nucleus_info(
+        cls: type[Self],
+        z: int,
+        a: int,
+        anti: bool = False,
+        i: int = 0,
+        l_strange: int = 0,
+    ) -> Self:
+        """
+        Get a nucleus particle from the neutron and proton count.
+        As described here https://pdg.lbl.gov/2007/reviews/montecarlorpp.pdf:
+        "To avoid ambiguities,nuclear codes should not be applied to a single hadron, like p, n
+        or Λ0, where quark-contents-based codes already exist."
```

Comment:
```suggestion
        or the Λ, where quark-contents-based codes already exist."
```

Reply:
I did not check - by this you mean that the function does not work for a proton, which can be interpreted as a hydrogen nucleus?